build.gradle.kts

```diff
@@ -0,0 +1,89 @@
+import java.io.ByteArrayOutputStream
+
+plugins {
+    groovy
+    `java-gradle-plugin`
+    kotlin("jvm") version "1.4.32"
+    id ("com.gradle.plugin-publish") version "0.14.0"
+}
+
+repositories {
+    mavenCentral()
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+pluginBundle {
+    website = "https://github.com/jsonschema2dataclass/js2d-gradle"
+    vcsUrl = "https://github.com/jsonschema2dataclass/js2d-gradle.git"
+    tags = listOf("json-schema", "jsonschema", "generator", "pojo", "jsonschema2pojo",
+            "dataclass", "data", "json", "generation", "jsonschema2dataclass", "java")
+}
+
+gradlePlugin {
+    plugins {
+        create("jsonschema2dataclassPlugin") {
+            id = "org.jsonschema2dataclass"
+            implementationClass = "com.github.js2d.JsonSchemaPlugin"
+            displayName = "Extended Gradle JsonSchema2Pojo Plugin"
+            description = "A plugins that generates Java sources from JsonSchema using jsonschema2pojo. Please, see the GitHub page for details"
+            version = getTag()
+        }
+    }
+}
+
+dependencies {
+    implementation ("org.codehaus.groovy:groovy-all:3.0.8")
+    implementation( "org.jsonschema2pojo:jsonschema2pojo-core:1.1.1")
```

Comment:
Super minor issue. I guess the formatting is not right

Reply:
Could you please tell me which formatting is correct for `build.gradle.kts` and have my `~/.editorconfig.conf` set for 4 spaces by default

Reply:
Simply use 
<img width="313" alt="image" src="https://user-images.githubusercontent.com/3314607/116030913-f0145b00-a65c-11eb-8fd2-029e78b9d3cd.png">

Reply:
ok, I pressed wrong key combination :) But numbers are important as they override default IntelliJ settings

Reply:
> have my ~/.editorconfig.conf set for 4 spaces by default

The word **_my_** is wrong here. It should be common, please see #102 

Here is the codestyle defined
https://github.com/jsonschema2dataclass/js2d-gradle/pull/102/files#diff-6c3e13b4e32c149ec42839ac3da504c9ea988311698c5a59535556a21fac4825R3

Reply:
I agree. There's hierarchy how IntelliJ IDEA reads formatting settings:

common IntelliJ IDEA settings
global editor config settings
project IntelliJ IDEA settings & project editor config settings.

Editor config is a common denominator between multiple editors

I prefer project-specific editor config settings rather than IntelliJ IDEA's settings if we're talking about margins and spaces as I use MacVim often and it's hard to synchronize settings manually or open IntelliJ IDEA just for formatting. 

I hope you understand me

Reply:
My editor config you can find [here](https://bitbucket.org/eirnym/.vim/raw/d8819ebe954a20167355b91ae95c0572b71e3f3b/.editorconfig)

Reply:
> I prefer project-specific editor config settings rather than IntelliJ IDEA's settings

It is (or would be) ! https://github.com/jsonschema2dataclass/js2d-gradle/pull/102/files#diff-6c3e13b4e32c149ec42839ac3da504c9ea988311698c5a59535556a21fac4825R3

Reply:
Indeed, common code format over different tools is not that easy

Reply:
I at least prefer a way, where I clone repository, open it and all is fine.
No additional step should be required

Reply:
So I prefer keep as many things as possible in editor config which is common for every editor

Reply:
https://editorconfig.org

Reply:
there's even [gradle plugin](https://github.com/ec4j/editorconfig-gradle-plugin) to check complacence

Reply:
ok, it should be correct now, let's continue this conversation in a separate PR you created